### PR TITLE
Code docs will be required on all PR's when this PR is closed

### PR DESCRIPTION
**Pull request by optimumtact**
This PR will be closed in 5 days.

You can find the guide to use comments to autodoc your code here
https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md

if you're going to make a PR, you should autodoc it anyway.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,3 +46,5 @@ The TGS3 API is licensed as a subproject under the MIT license.
 See the footers of code/\_\_DEFINES/server\_tools.dm, code/modules/server\_tools/st\_commands.dm, and code/modules/server\_tools/st\_inteface.dm for the MIT license.
 
 All assets including icons and sound are under a [Creative Commons 3.0 BY-SA license](https://creativecommons.org/licenses/by-sa/3.0/) unless otherwise indicated.
+
+IF you read this, you are the big dum dum :P
```
Comment (tralezab) [inline README.md]: this is not so correct

**Review by tralezab — COMMENTED** [review]
one small error you'll have to fix 😋

**Review by nicbn — COMMENTED** [review]
no u

**Comment by ExcessiveUseOfCobblestone** [comment]
should put this link (and the requirement to document in this format) in contributing.md

**Comment by trollbreeder** [comment]
When and where should you have to add autodocumentation?
1. A PR that fixes a bug, but does nothing else? (For example, a bug with reagents)
2. A PR that adds a new thing to an existing module? (For example, a new reagent)
3. A PR that adds a new module? (For example, a new reagent system)

**Comment by ExcessiveUseOfCobblestone** [comment]
example would be a new reagent

```DM
/datum/reagent/cute-iver //no doc req, reagent should be doc'd and not using anything unfamiliar to parent
    name = "cute-iver" //no doc req, should be doc'd where 'var/'d
    ...
    var/iscute = FALSE //doc req, new var
```
then uh oh cobby forgot to make it heal
```DM
/datum/reagent/cute-iver/on_mob_metabolize(mob/living/L) //no doc req, should be doc'd where 'proc/'d
    L.adjustToxLoss(3,TRUE) >> L.adjustToxLoss(-3,TRUE) // no doc req, changed to make it work as intended and not adding anything new
```
but what if I want to add a physical properties datum for reagents
```DM
/datum/reagent_properties // doc req, new class
    var/name = "" // doc req, new var
    ....

/datum/reagent_properties/proc/DoTheThing(atom/A) //doc req, new proc
```

**Comment by trollbreeder** [comment]
thanks for the clarification cobby

**Comment by optimumtact** [comment]
perhaps you should read the document linked before asking questions already covered.

**Comment by ATH1909** [comment]
What if a PR is made without autodoc before this PR is closed, but isn't merged by the time that this PR is closed?

File: README.md (hunk 1 above)
Comment (mrhugo13) [inline README.md]: no you

**Comment by optimumtact** [comment]
Only things that are open after this closes.

But I urge you to autodoc your code, even now, people should have been doing it weeks ago.

It only takes a few moments when writing a proc.

**Pull request closed without merging**